src/charm.py

```diff
@@ -42,32 +42,32 @@ def _on_install(self, _: ops.InstallEvent) -> None:
         try:
             self.bind.prepare()
             self.unit.status = ops.ActiveStatus()
-        except exceptions.BlockableError as e:
-            self.unit.status = ops.BlockedStatus(e.msg)
+        except exceptions.SnapError as e:
+            self.unit.status = ops.ErrorStatus(e.msg)
```

Comment:
Does this work? https://ops.readthedocs.io/en/latest/#ops.ErrorStatus says that ErrorStatus should not be set. Maybe just not catch the error (or catch, log and reraise, although I suspect a stack trace will be logged by ops anyway).

Reply:
Good point. Fixed